index.html

```diff
@@ -229,5 +229,118 @@ <h2 class="text-4xl mb-8 font-semibold">
         >
       </div>
     </div>
+    
+ <!-- new card in a container -->
+    
+    <div class="container mx-auto py-24 px-5">
+    <div class=" grid grid-cols-1 gap-5 sm:grid-cols-1 md:grid-cols-1 lg:grid-cols-1 xl:grid-cols-2">
+      
+      <!-- card 1  -->
+
+      <div class="bg-gray-0 h-32 md:h-48 lg:h-48">
+        <div class="flex flex-inline ">
+          <div class=" h-32 w-32 lg:h-48 lg:w-48 md:h-48 md:w-48 ">
+            <img class="rounded-full" src="https://images.unsplash.com/photo-1535713875002-d1d0cf377fde?crop=entropy&cs=tinysrgb&fit=crop&fm=jpg&h=500&ixid=eyJhcHBfaWQiOjF9&ixlib=rb-1.2.1&q=80&w=500" alt="">
+          </div>
+          <div class=" lg:w-2/3 md:w-2/3 sm:w-2/3  px-5 pt-5 sm:pt-2 lg:pt-5 rounded-r-lg">
+            <button class="bg-yellow-400 rounded px-1 text-white text-sm ml-auto"> community </button>
+            <h1
+              class="font-medium text-lg sm:text-xl md:text-xl lg:text-xl xl:text-xl font-medium pt-2 sm:pt-1 md:pt-1 lg:pt-1">
+              Eddie Jaoude</h1>
+            <p
+              class="font-light text-xs sm:text-sm md:text-sm lg:text-sm xl:text-sm py-1 font-normal hidden   md:block lg:block break-words">
+              Fullstack open source DevRel! I love code and the community :). GitHub Star ⭐ program 🤓</p>
+            <div class=" hidden lg:inline-flex  md:inline-flex  flex flex-inline">
+              <div class="flex flex-inline">
+                <i class="fa fa-github-alt pr-1 pt-6 "
+                  style="font-size:18px; color:rgb(109, 71, 199); font-style:normal;"></i> </i>
```

Comment:
These should be tailwind styles

Reply:
@eddiejaoude okay, brother now where do I make these changes ??

Reply:
New PR please 🤓